src/codegen/enums.rs

```diff
@@ -24,6 +24,7 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
     let mut has_get_quark = false;
     let mut has_any = false;
     let mut has_get_type = false;
+    let mut generate_display_trait = true;
```

Comment:
IMHO this better be "= false":
If no enums then don't need add "use std::fmt",
also assignment will always "true"

Reply:
Oh right! I had the fact that by default everything has an implemention of Display... Good catch!